tasks/section2.yml

```diff
@@ -396,11 +422,12 @@
       - patch
       - rule_2.2.7
 
-- name: "SCORED | 2.2.7 | PATCH | Ensure NFS and RPC are not enabled"
+- name: "SCORED | 2.2.7 | PATCH | Ensure RPC is not enabled"
   service:
       name: rpcbind
       state: stopped
       enabled: no
+  changed_when: no
```

Comment:
This `changed_when: no` line shouldn't be needed anymore.

Reply:
:full_moon:

Reply:
Go ahead and try it..